src/components/layout/besquare/footer.tsx

```diff
@@ -63,15 +63,15 @@ const BeSquareFooter = () => {
                         alt="Fresh graduates discussing with their laptops"
                     />
                     <TextContainer>
-                        <Title as="h3">Next intake: 13 June 2022</Title>
+                        <Title as="h3">Next intake: January 2023</Title>
                         <TextWrapper
                             max_width={['510px', '328px']}
                             font_size={['20px', '16px']}
                             line_height={['30px', '24px']}
                             margin={['-24px 49px 0 0']}
                         >
                             <Text color="white" size="2rem">
```

Comment:
Hi @konstantinosG-deriv , it was advised that Text from styled component is depricated and you can use Header instead.